### PR TITLE
cmake format

### PR DESCRIPTION
**Pull request by neheb**

### CODE REVIEW

**Comment by neheb** [comment]
This CI is not very good. It's picking up non cmake files. It also makes no sense to lint third party ones.

File: app/CMakeLists.txt [hunk 1]
```diff
@@ -1,57 +1,62 @@
...
 endif()
 
 target_link_libraries(exiv2 PRIVATE std::filesystem)
 
 if(MSVC OR MINGW)
-    # Trick to get properly UTF-8 encoded argv 
-    # More info at: https://github.com/huangqinjin/wmain
-    add_library(wmain STATIC wmain.cpp)
-    target_link_libraries(exiv2 PRIVATE wmain)
+  # Trick to get properly UTF-8 encoded argv More info at: https://github.com/huangqinjin/wmain
```
Comment (kmilos) [inline app/CMakeLists.txt:30]: Comment unwrapped

File: cmake/exiv2.pc.in [hunk 2]
```diff
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=@libdir_for_pc_file@
-includedir=@includedir_for_pc_file@
+libdir=@LIBDIR@
+includedir=@INCLUDEDIR@
```
Comment (kmilos) [inline cmake/exiv2.pc.in:4]: These shortened variables are too generic and there is risk they are used by some other project that could e.g. include the Exiv2 one...

File: app/CMakeLists.txt [hunk 3]
```diff
@@ -1,57 +1,62 @@
-add_executable( exiv2
-     exiv2.cpp
-     exiv2app.hpp
-     actions.cpp actions.hpp
-     getopt.cpp getopt.hpp
-     app_utils.cpp app_utils.hpp
+add_executable(
+  exiv2
```
Comment (kmilos) [inline app/CMakeLists.txt:2]: Is there a way to keep the target separate from the sources list, i.e. still on the first line, or have sources indented more?

File: po/CMakeLists.txt [hunk 4]
```diff
@@ -1,38 +1,34 @@
...
-    pt.po
-    ru.po
-    sk.po
-    sv.po
-    ug.po
-    uk.po
-    vi.po
+gettext_create_translations(
+  exiv2.pot
+  ALL
```
Comment (kmilos) [inline po/CMakeLists.txt:5]: Ditto re target vs sources list.

File: app/CMakeLists.txt (hunk 1 above)
Comment (neheb) [reply app/CMakeLists.txt]: Added a newline. Linter is happy now.

File: cmake/exiv2.pc.in (hunk 2 above)
Comment (neheb) [reply cmake/exiv2.pc.in]: I disagree but renamed. It's unfortunate CMake doesn't have a pkgconfig writer like meson's.

File: app/CMakeLists.txt (hunk 3 above)
Comment (neheb) [reply app/CMakeLists.txt]: The way to do it is to have a separate variable. Did that.

File: po/CMakeLists.txt (hunk 4 above)
Comment (neheb) [reply po/CMakeLists.txt]: Same as above.

**Review by kmilos — APPROVED** [review]
LGTM

**Pull request merged**